package.json

```diff
@@ -1,76 +1,66 @@
 {
-  "name": "@skyux/autonumeric",
-  "version": "5.0.0-beta.0",
-  "description": "Library for skyux-lib-autonumeric",
+  "name": "autonumeric-workspace",
+  "version": "0.0.0",
   "scripts": {
-    "build": "skyux build-public-library",
-    "test": "skyux test",
-    "test:watch": "skyux watch",
-    "start": "skyux serve",
-    "e2e": "skyux e2e",
-    "pact": "skyux pact",
-    "pact:watch": "skyux pact --watch",
-    "lint": "skyux lint"
-  },
-  "keywords": [],
-  "author": "Blackbaud",
-  "license": "ISC",
-  "repository": {
-    "type": "git",
-    "url": ""
-  },
-  "peerDependencies": {
-    "@angular/common": "^9.1.6 || ^12",
-    "@angular/core": "^9.1.6 || ^12",
-    "@angular/forms": "^9.1.6 || ^12"
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test",
+    "skyux:generate-lib-resources-module": "ng generate @skyux/i18n:lib-resources-module",
+    "lint": "ng lint",
+    "skyux:generate-documentation": "ng generate @skyux-sdk/documentation-schematics:documentation",
+    "postbuild": "npm run skyux:generate-documentation"
   },
+  "private": true,
   "dependencies": {
-    "autonumeric": "4.6.0"
```

Comment:
☝️ Note: `ng build` was giving me warnings that libraries should not have `dependencies`. Moved to `peerDependencies`.

Reply:
Disregard. This is a breaking change. Reverting back to `dependencies`.